examples/variables.py

```diff
@@ -0,0 +1,63 @@
+# Enablement of https://argoproj.github.io/argo-workflows/variables/
+# This example creates two tasks, one of the Tasks is a deamond task and its IP address is shared with the second task
+# The daemoned task operates as server, serving an example payload, with the second task operating as a client, making
+# http requests to the server
+
+from hera.task import Task
+from hera.input import InputParameterAsEnv
+from hera.workflow import Workflow
+from hera.workflow_service import WorkflowService
+
+
+def server():
+    pass
+
+
+def consumer(server_ip):
+    print(server_ip)
+
+
+# TODO: replace the domain and token with your own
+ws = WorkflowService(host='my-argo-server.com', token='my-auth-token')
+
+
+def server():
+    from http.server import BaseHTTPRequestHandler, HTTPServer
+
+    class MyServer(BaseHTTPRequestHandler):
+        def get_check(self):
+            self.send_response(200)
+            self.send_header("Content-type", "application/json")
+            self.end_headers()
+            self.wfile.write(bytes("{'name':'John'}", "utf-8"))
+
+    webServer = HTTPServer(("0.0.0.0", 8080), MyServer)
+    try:
+        webServer.serve_forever()
+    except KeyboardInterrupt:
+        pass
```

Comment:
This is a very interesting example! I do not have experience with daemon tasks - how does this task know when to be "done"? 🙂

Reply:
I cleaned this up as it implied something incorrect - Argo simply shuts down the daemon once the other Tasks have been completed.

Reply:
Ah, I see, so marking something as a daemon keeps the pod alive but Argo terminates the pod after it's done 👍🏻